portfolio/src/main/webapp/script.js

```diff
@@ -76,7 +76,8 @@ document.addEventListener('DOMContentLoaded', () => {
 });
 
 document.addEventListener('DOMContentLoaded', () => {
-  checkLoginStatus();
+  document.getElementById('login-status').addEventListener('click', login);
+  login();
```

Comment:
did you intend this change to be here? looks like you call login immediately and when the user clicks the login-status element

Reply:
Thank you!